app/controllers/course/forum/posts_controller.rb

```diff
@@ -0,0 +1,56 @@
+class Course::Forum::PostsController < Course::Forum::Controller
+  include Course::Discussion::PostsConcern
+
```

Comment:
I think it should be before, but you can go try both. I prefer declarations rather than imperative operations.